src/Tasks/Microsoft.CSharp.targets

```diff
@@ -17,25 +17,25 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <!--
-        In VS 2010 SP1 and VS 2012, both supported for asset compatibility, the MSBuild installed 
-        as part of them did not enforce using the local ToolsVersion (4.0) in all cases, but instead 
-        just used whatever ToolsVersion was in the project file if it existed on the machine, and 
-        only forced 4.0 if that ToolsVersion did not exist.  
```

Comment:
I don't know why VSCode did all these whitespace changes, but they seem better than what was there initially.

Reply:
True, but it also makes it harder to figure out what you actually changed. I saw you split Microsoft.Managed.targets into Microsoft.Managed.Before.targets and Microsoft.Managed.After.targets, both as far as references to them and the code, and that you (conditionally) modified _MainReferenceTarget instead of _RecursiveTargetForContentCopying. Were those the only changes?

Reply:
Yup, I split the target file in two, because some things need to be set before the common targets (common targets read them), and some things after (common targets compute stuff read by them)